test/Interop/Cxx/templates/Inputs/function-templates.h

```diff
@@ -61,6 +61,8 @@ template <class T> void constLvalueReference(const T &) {}
 
 template <class T> void forwardingReference(T &&) {}
 
+template <class T> void PointerTemplateParameter(T*){}
```

Comment:
If you want you could add a few more tests, such as a method or a function template with multiple parameters.

But this is probably OK-- it's not a very big change.